pkg/ctl/create/iamidentitymapping.go

```diff
@@ -40,35 +25,37 @@ func createIAMIdentityMappingCmd(cmd *cmdutils.Cmd) {
 		`),
 	)
 
-	var options iamIdentityMappingOptions
-
 	cmd.CobraCommand.RunE = func(_ *cobra.Command, args []string) error {
 		cmd.NameArg = cmdutils.GetNameArg(args)
-		return doCreateIAMIdentityMapping(cmd, options)
+		return doCreateIAMIdentityMapping(cmd)
 	}
 
+	cfg.IAMIdentityMappings = []*api.IAMIdentityMapping{{}}
+	cmd.FlagSetGroup.InFlagSet("IAMIdentityMapping", func(fs *pflag.FlagSet) {
+		fs.StringVar(&cfg.IAMIdentityMappings[0].Account, "account", "", "Account ID to automatically map to its username")
+		cmdutils.AddIAMIdentityMappingARNFlags(fs, cmd, &cfg.IAMIdentityMappings[0].ARN, "create")
+		fs.StringVar(&cfg.IAMIdentityMappings[0].Username, "username", "", "User name within Kubernetes to map to IAM role")
+		fs.StringSliceVar(&cfg.IAMIdentityMappings[0].Groups, "group", []string{}, "Groups within Kubernetes to which IAM role is mapped")
+		fs.StringVar(&cfg.IAMIdentityMappings[0].ServiceName, "service-name", "", "Service name; valid value: emr-containers")
+		fs.StringVar(&cfg.IAMIdentityMappings[0].Namespace, "namespace", "", "Namespace in which to create RBAC resources (only valid with --service-name)")
+		fs.BoolVar(&cfg.IAMIdentityMappings[0].NoDuplicateArns, "no-duplicate-arns", false, "Throw error when an aws_auth record already exists with the given arn.")
```

Comment:
```suggestion
		fs.BoolVar(&cfg.IAMIdentityMappings[0].NoDuplicateArns, "no-duplicate-arns", false, "Throw error when an aws-auth record already exists with the given ARN.")
```

I know this wasn't introduced by your PR.